dockerfiles/hpcc/templates/datastorage.yaml

```diff
@@ -1,28 +1,17 @@
-apiVersion: v1
```

Comment:
we should rename datastorage.yaml to datastorage-claim.yaml (to be consistent with dllserver-claim.yaml)

Reply:
we should rename them both - I have a separate Jira https://track.hpccsystems.com/browse/HPCC-23649 in progress where I intend to address that.